tgui/packages/tgui/interfaces/XenomorphExtractor.jsx

```diff
@@ -0,0 +1,144 @@
+import { useBackend, useLocalState } from '../backend';
+import { Section, Button, Stack, NoticeBox, LabeledList, Flex, Box, Dropdown, Divider } from '../components';
+import { Window } from '../layouts';
+
+export const XenomorphExtractor = (_props, context) => {
+  const { act, data } = useBackend(context);
+
+  const {
+    organ,
+    points,
+    upgrades,
+    caste,
+    value,
+    categories,
+    current_clearance,
+    is_x_level,
+  } = data;
+  const dropdownOptions = categories;
+  const [selectedTab, setSelectedTab] = useLocalState('category', 'NONE');
+
+  return (
+    <Window width={850} height={800} theme="crtyellow">
+      <Window.Content scrollable>
+        <Section>
+          <Stack fill vertical>
+            <Stack.Item>
+              <Button
+                fluid
+                icon="eject"
+                content={
+                  !organ ? 'Eject Biomass' : 'Eject ' + caste + ' biomass'
+                }
+                disabled={!organ}
+                onClick={() => act('eject_organ')}
+              />
+            </Stack.Item>
+            <Stack.Item>
+              <Button
+                fluid
+                icon="eject"
+                content={
+                  !organ
+                    ? 'Process Biomass'
+                    : 'Process Biomass, Expected value : ' + value
+                }
```

Comment:
Just a heads up that after https://github.com/cmss13-devs/cmss13/pull/6244 buttons should not use the content property. Instead their content needs to be as a child e.g.

```js
<Button content="Bioscan" />
```

Needs to instead be:
```js
<Button>Bioscan</Button>
```